src/python/pants/bin/local_pants_runner.py

```diff
@@ -234,7 +233,7 @@ def _run_inner(self) -> ExitCode:
         try:
             return self._perform_run(goals)
         except Exception as e:
-            ExceptionSink.log_exception(e)
+            logger.error(e)
```

Comment:
Our installed `logger` is aware of the `print-stacktrace` setting, but the `ExceptionSink` is not. The `ExceptionSink.log_exception` method is also only intended for uncaught exceptions, and this case is clearly caught.